src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -260,13 +262,34 @@ define([
                         this.options.fields.description,
                         this.options.separators.description
                     ),
-                    isVideo = (item.type.name === 'video');
+                    isVideo = (item.type.name === 'video'),
+                    deferred = $.Deferred(),
```

Comment:
Do we really need deferreds here? Does the dropdown handle something in async here?